src/axom/primal/tests/primal_boundingbox.cpp

```diff
@@ -699,6 +730,37 @@ TEST(primal_boundingBox, bb_get_centroid)
   PointType centroid = bbox.getCentroid();
   EXPECT_DOUBLE_EQ(0.5, centroid[0]);
   EXPECT_DOUBLE_EQ(0.5, centroid[1]);
+
+  // The centroid of an invalid bbox is 0,0.
+  BoxType bbox2;
+  EXPECT_DOUBLE_EQ(bbox2.getCentroid()[0], 0.);
+  EXPECT_DOUBLE_EQ(bbox2.getCentroid()[1], 0.);
```

Comment:
Looks like this was the previous behavior as well (?)

Reply:
Yes, that's right. I thought that is what it would do but was checking.

Reply:
If an invalid bbox is centered at 0, then it's actually kind of surprising to me that expand and shift wouldn't move it around.